apps/modernization-ui/src/apps/search/investigation/InvestigationSearch.tsx

```diff
@@ -26,11 +27,16 @@ const defaultValues: InvestigationFilterEntry = {
 
 const InvestigationSearch = () => {
     const { status, search, reset, results } = useInvestigationSearch();
+    const { sorting } = useSorting();
     const form = useForm<InvestigationFilterEntry, Partial<InvestigationFilterEntry>>({
         mode: 'all',
         defaultValues
     });
 
+    useEffect(() => {
+        sorting && search(form.getValues());
+    }, [sorting]);
+
```

Comment:
suggestion:  Instead of copying this in every Search component add it to `useSearch`